### ISSUE TITLE
Deprecate Travis CI in favor of CircleCI

### ISSUE THREAD

**Issue by jrgriffiniii** [pulibrary/orangelight#1116]
Currently attempting to [update Ruby to release 2.5.0](https://github.com/pulibrary/orangelight/issues/914)  is blocked by [an unrelated Travis CI bug](https://github.com/travis-ci/travis-ci/issues/8969).  Given that there are other repositories currently using CircleCI for continuous integration testing, using this service may permit the update to 2.5.0 to be undertaken.

**Comment by jrgriffiniii** [comment]
Following a discussion held on 01/09/18, no one objected to this measure, but it was held as a lower priority due to the following concerns: https://github.com/pulibrary/orangelight/issues/1112#issuecomment-356377225
